.circleci/config.yml

```diff
@@ -26,7 +24,7 @@ jobs:
           paths:
             - ~/.cache/pre-commit
       - save_cache:
-          key: go-mod-sources-v2-{{ checksum "go.sum" }}
+          key: go-mod-sources-v2-{{ checksum "go.sum" }}-{{ checksum "scripts/check-go-version" }}
```

Comment:
```suggestion
          key: go-mod-sources-v2-{{ checksum "go.sum" }}-{{ checksum "bin/check_go_version" }}
```

Reply:
If the path was wrong why did the error go away & the imports update?

Reply:
I think because the cache was just broken --https://app.circleci.com/pipelines/github/trussworks/truss-aws-tools/22/workflows/1e10a557-363e-45bc-bfe2-5db8f9ece35f/jobs/488/steps

Reply:
Have I fixed this or did I just make it fubar?